### ISSUE TITLE
usb/storage-preinserted-disk fails with uc24

### ISSUE THREAD

**Issue by plars** [canonical/checkbox#1209]
### Bug Description

I confirmed that there's a usb disk in the device, and from an ssh session to the DUT I can even mount it with no errors. However when the checkbox test runs, I get the following errror. Perhaps it's a confinement issue?
```
[ Test USB storage on 2.0 or 1.1 ports detected by udev (disk/by-uuid/64C6-EAAD) ]
ID: com.canonical.certification::usb/storage-preinserted-disk/by-uuid/64C6-EAAD
Category: USB tests
--------------------------------------------------------------------------------
DEBUG:generating a random file
DEBUG:try to mount usb storage for testing
mount: /tmp/tmp438evm7r: permission denied.
       dmesg(1) may have more information after failed mount system call.
ERROR:mount /dev/disk/by-uuid/64C6-EAAD on /tmp/tmp438evm7r failed.
INFO:context manager exit: unmount USB storage
umount: /tmp/tmp438evm7r: not mounted.
WARNING:umount /tmp/tmp438evm7r failed.
INFO:Remove temporary folders and files.
--------------------------------------------------------------------------------
Outcome: job failed
```

### To Reproduce

run checkbox tests on a rpi with uc24 beta installed and a usb disk inserted

### Environment

rpi 4/5 at least, possibly others
Ubuntu Core 24 (current beta image)

**Comment by plars** [comment]
Good news! I guess it was something in one of the snaps that just landed today in the UC24 image. Consoleconf, core24, and snapd all got updated and it works now! Closing this one since it doesn't seem to be a test issue

**Comment by plars** [comment]
Closed since it works as of 20240424 image
